__init__.py

```diff
@@ -355,10 +362,15 @@ def after_load(_a, _b):
 
 
 def gameEditorUpdate(self, context):
+    world_defaults = {}
     if self.gameEditorMode == "SM64":
-        self.f3d_type = "F3D"
+        self.f3d_type, world_defaults = "F3D", sm64_world_defaults
     elif self.gameEditorMode == "OOT":
-        self.f3d_type = "F3DEX2/LX2"
+        self.f3d_type, world_defaults = "F3DEX2/LX2", oot_world_defaults
+    elif self.gameEditorMode == "Homebrew":
+        self.f3d_type, world_defaults = "F3D", {}  # This will set some pretty bad defaults, but trust the user
+    if self.world:
```

Comment:
did you mean ``if self.world is not None`` or ``if len(self.world) > 0``? I think we should avoid doing ``if var`` it can bring a lot of confusion imo